app/femr/data/models/mysql/keys/PatientKey.java

```diff
@@ -0,0 +1,71 @@
+package femr.data.models.mysql.keys;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Embeddable
+public class PatientKey implements Serializable {
+    @Column(name = "id", unique = true, nullable = false)
+    private int patientId;
+    @Column(name = "kit_id", nullable = false)
+    private int kitId;
+
+    public PatientKey(int patientId, int kitId) {
+        this.patientId = patientId;
+        this.kitId = kitId;
+    }
+
+    public PatientKey() {}
```

Comment:
is this needed?

Reply:
To create a compound key in ebean, and embeddable key is required. This is the class for the embeddable key which represents a patients primary compound key. So I would say this is needed. The other option for a compound key is to do something like @IdClass, but ebean doesn't support that sadly. If it did, that would have been way easier because it wouldn't require a new class and key object.